ui/stories/ttl-picker2.md

```diff
@@ -0,0 +1,31 @@
+<!--THIS FILE IS AUTO GENERATED. This file is generated from JSDoc comments in app/components/ttl-picker2.js. To make changes, first edit that file and run "yarn gen-story-md ttl-picker2" to re-generate the content.-->
+
+## TtlPicker2
+TtlPicker2 components are used to enable and select time to live values. Use this TtlPicker2 instead of TtlPicker if you:
+- Want the TTL to be enabled or disabled
+- Want to have the time recalculated by default when the unit changes (eg 60s -> 1m)
+
+**Params**
+
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| onChange | <code>function</code> |  | This function will be passed a TTL object, which includes enabled{bool}, seconds{number}, timeString{string}. |
+| [label] | <code>string</code> | <code>&quot;&#x27;Time to live (TTL)&#x27;&quot;</code> | Label is the main label that lives next to the toggle. |
+| [helperTextDisabled] | <code>string</code> | <code>&quot;&#x27;Allow tokens to be used indefinitely&#x27;&quot;</code> | This helper text is shown under the label when the toggle is switched off |
```

Comment:
this is a problem with our `gen-story-md` script, but the `&quot;&#x27;` are showing up 😬 
<img width="528" alt="Screen Shot 2020-04-02 at 12 42 30 PM" src="https://user-images.githubusercontent.com/903288/78294606-810f1600-74df-11ea-9878-3b7de42dbd91.png">